### ISSUE TITLE
Content of file.d.ts not injected in dist/index.d.ts

### ISSUE THREAD

**Issue by gmpetrov** [jaredpalmer/tsdx#174]
Hello, I'm new to typescript, and I don't understand what is the good way to declare an interface for it to be exported within the build.

### Current Behavior

In src/global.d.ts, I have interfaces declared as:

interface Hello {
  world: string;
}

After the build, dist/index.d.ts does not contain interface Hello.

### Expected behavior

After the build, dist/index.d.ts is containing interface Hello.

**Comment by agilgur5** [comment]
Duplicate of #113 .

`tsc` doesn't re-output declaration files, and neither does TSDX or `rollup-plugin-typescript2` (which TSDX uses) currently (see #113 for the feature request).
You can read more about it that at this [StackOverflow Question](https://stackoverflow.com/q/56018167/3431180) and this issue: https://github.com/ezolenko/rollup-plugin-typescript2/issues/166#issuecomment-521315573

tl;dr is that if you want/need declarations in `dist/`, you should copy them over.
